src/ostorlab/serve_app/types.py

```diff
@@ -735,10 +718,10 @@ class OxoLinkInputType(graphene.InputObjectType):
 
 
 class OxoAssetInputType(graphene.InputObjectType):
-    android_file = OxoAndroidFileAssetInputType()
-    ios_file = OxoIOSFileAssetInputType()
-    android_store = OxoAndroidStoreAssetInputType()
-    ios_store = OxoIOSStoreAssetInputType()
+    android_apk_file = graphene.List(OxoAndroidFileAssetInputType)
+    ios_file = graphene.List(OxoIOSFileAssetInputType)
+    android_store = graphene.List(OxoAndroidStoreAssetInputType)
+    ios_store = graphene.List(OxoIOSStoreAssetInputType)
```

Comment:
Link and IP? are the names corrects?

Reply:
here are the names: 
![image](https://github.com/Ostorlab/oxo/assets/144013278/d785e79f-ddb4-444a-b0cb-e92746454409)